netplugin/netd.go

```diff
@@ -456,6 +456,18 @@ func createContainerEpOper(netPlugin *plugin.NetPlugin, contUUID, contName strin
 				log.Errorf("Endpoint creation failed. Error: %s", err)
 				return err
 			}
+
+			err = operEp.Read(epID)
```

Comment:
The issue was caught with the new sanity test cases I added where containers are deleted and recreated. On a delete the operstate is completely wiped out. So on a recreate - 439 read is a failure. CreateEndpoint creates the operstate but ContUUID is not added as a part of this. Hence the explicit read to read the operstate and populating the ContID